src/main/java/duke/ui/Greetings.java

```diff
@@ -0,0 +1,66 @@
+package duke.ui;
+
+public class Greetings {
```

Comment:
Class name seems to be off since you have a lot than Greeting. You might consider to change the name